### PR TITLE
new-feature:add k3wsjobd

### PR DESCRIPTION
**Pull request by IloveKanade**
The k3wsjobd modified according to the specification

### CODE REVIEW

File: wsjobd.py [hunk 1]
```diff
@@ -0,0 +1,373 @@
+#!/usr/bin/env python2
```
Comment (drmingdrmer) [inline wsjobd.py:1]: We do not need python2 anymore~

**Review by drmingdrmer — COMMENTED** [review]
CI does not pass yet 🤔

File: wsjobd.py (hunk 1 above)
Comment (IloveKanade) [reply wsjobd.py]: done~~~

**Comment by IloveKanade** [comment]
> CI does not pass yet 🤔

done~

**Review by drmingdrmer — APPROVED** [review]
LGTM!

**Pull request merged**